src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -647,6 +727,19 @@ async function ProcessBackground(
     }
   }
 
+  if (service === "firebaseio.com") {
```

Comment:
nit: we could use the service constants here: https://github.com/firebase/firebase-tools/blob/master/src/emulator/constants.ts#L15-L16